js/scenes/SurveyComplete/SurveyComplete.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react'
+import { Text, View } from 'react-native';
+
+import { styles } from './styles';
+import SurveyButton from '../../components/SurveyButton';
+import { returnToHome, returnToEvents, returnToSpeaker } from '../../navigation/navHelpers';
+
+class SurveyCompleteContainer extends Component {
+
+  static route = {
```

Comment:
If not using internal `state` or life cycle methods, please use functional components.

Reply:
Unfortunately, our ExNavigator library requires standalone scenes to be classes so that we can attach the static route options to them.